docsrc/generic-doc.fsx

```diff
@@ -136,7 +136,7 @@ For a type defined in an external library it will work when it contains a static
 
 Here's an example of the generic function ``fromBigInt`` targeting a type defined in the MathNet library
 *)
-#r "../../packages/docs/MathNet.Numerics/lib/net40/MathNet.Numerics.dll"
-#r "../../packages/docs/MathNet.Numerics.FSharp/lib/net45/MathNet.Numerics.FSharp.dll"
+#r "../packages/docs/MathNet.Numerics/lib/net40/MathNet.Numerics.dll"
```

Comment:
these are added through download nuget scripts, so by deleting the tools folder you will need to download these another way.